### ISSUE TITLE
isinstance("",str) breaks on Python2 after running futurize

### ISSUE THREAD

**Issue by avim2809** [PythonCharmers/python-future#560]
i have a very big project which i am trying to conver to py3 but keep it working on py2 in the meanwhile 
after running futurize it breaks string type checks 
from builtins import str
assert isinstance("s",str)

according to the documentation on your website  (https://python-future.org/isinstance.html?highlight=isinstance#isinstance) 
there would be no alternative but using unicode_literals to avoid this

but it doesnt make sense since everywere i read i see many against it 

so bottom line is there a way to avoid this issue without unicode_literals

**Comment by ngie-eign** [comment]
@avim2809 : we ran into similar with our group -- the trick is to use `six.string_types` which will allow `isinstance` to check the string type properly.
